.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        deno: ["v2.x", "canary"]
+        deno: ["v2.x", "v2.0.x", "canary"]
```

Comment:
I can remove this change too... was just troubleshooting why CI was failing for unrelated tests. 

Indeed it seems to be a 2.1 problem. GH issue here: https://github.com/denoland/fresh/issues/2771